Package.swift

```diff
@@ -99,55 +99,59 @@ let package = Package(
 
         // MARK: SwiftPM specific support libraries
 
+        .target(
+            name: "Core",
```

Comment:
I think I have somewhat of a preference for "Base" or "Utility" here, since to me, "Core" connotes the essence of the functionality of a component (e.g. core model and logic).  Maybe I'm just being pedantic though.